man/man1/oci-kvm.1

```diff
@@ -35,6 +35,24 @@ oci-kvm \- install and remove KVM guests on Oracle Cloud Infrastructure instance
 .B -p
 .I PATH}
 
+.B oci-kvm create-network -n|--net
+.I NET
+.B -N|--network-name
+.I NETWORK_NAME
+.B -v|--vcn-cidr
+.I VCN_CIDR
+.B -F|--ip-first
```

Comment:
the --ip-first is extremely confusing since there is a --ip-start as well, I understand this is the bridge IP, or guest gateway IP, maybe call this --ip-bridge or something, --ip-first just does not really make a lot of sense...

Reply:
changed to ip-bridge